Fix APR and APY not showing for aave (#5129)
rotki/rotki: docs/changelog.rst

Changelog
=========

* :bug:`5124` Users will now correctly see all the events related to lending in the defi view.

> * :bug:`5126` Users will see again their APR and APY for borrowing and lending in Aave.

Reviewer: ```suggestion
* :bug:`5126` APR and APY for borrowing and lending in Aave should properly show again.
```
